debian/clean

```diff
@@ -0,0 +1,3 @@
+include/swtpm/*.gch
+man/*/*.3
+man/*/*.8
```

Comment:
This cleans up files that otherwise weren't automatically picked up by the `clean` target.